x-pack/test/functional/page_objects/monitoring_page.js

```diff
@@ -25,6 +26,11 @@ export function MonitoringPageProvider({ getPageObjects, getService }) {
       await PageObjects.common.navigateToApp('monitoring');
     }
 
+    async getWelcome() {
+      const el = await find.byCssSelector('.euiCallOut--primary', 10000 * 10);
+      return await el.getVisibleText();
+    }
+
```

Comment:
This really isn't a monitoring method.  It was originally put here because it showed up on the Monitoring page a long time ago.  It's really part of telemetry but since telemetry is mostly behind the scenes code I don't think there's a page object for it.

Reply:
Should we move it?